### DISCUSSION TITLE
Issue in booting Linux image on Polarfire SoC discovery Kit

### DISCUSSION THREAD

**Discussion by Embedded-linux** [polarfire-soc/polarfire-soc-documentation#517] [Linux build systems for PolarFire SoC]
Hi all,

This is Satish, I have recently purchased board MPFS Discovery kit
Have programmed Linux on eNVM using reference file
https://github.com/polarfire-soc/polarfire-soc-discovery-kit-reference-design/releases/tag/2024.04

have copied boot images on SD card as below 
created 2 partitions
1 st part:   boot.scr  fitImage  uboot.env 
2nd part: core-image-minimal-xxx. wic  

part 1: W95  FAT system (0b)
part 2: Linux  (83)

mkfs.vfat /dev/sda1 -n boot
mkfs.ext4 /dev/sda2 -L root

but SD card is not identified by the board.
Have tried multiple options in file systems,still it is keep failing.

I am using pre built images provided by Microchip
where it is going wrong?
can you please suggest & provide the reference pages for Linux booting on Discovery kit.

Thanks in advance

Regards
Satish G

**Comment by griffini** [comment]
Duplicated in https://github.com/polarfire-soc/hart-software-services/issues/75 -- please follow the discussion there instead.
